### ISSUE TITLE
Consider lighter weight Docker image variant

### ISSUE THREAD

**Issue by shakuzen** [openzipkin-contrib/apache-release-verification#41]
Pulling the Docker image initially (and upon certain updates to the tag) requires downloading quite large layers (hundreds of megabytes). Have we considered using the Alpine Linux variant for the base image to reduce the size of layers? How difficult would it be to switch to that, and what tradeoffs would it involve?

/cc @abesto

**Comment by abesto** [comment]
Confession time: I put together the Docker image in the fastest way I possibly could. You're absolutely right it should be sitting on Alpine and be as small as possible (at which point also the wrapper script should pull before each run). I don't foresee any particular roadblocks with this (source: "Famous Last Words")

**Comment by shakuzen** [comment]
> Confession time: I put together the Docker image in the fastest way I possibly could.

Fair enough, and good approach, I think. I know Adrian and I at least sometimes are running on not the fastest public internet, so the big pulls can be painfully slow. If I find time, I will try to convert things to Alpine, or some other more lightweight variant. I'm not sure when I will likely have time to do this, though, so if anyone else has time and interest, feel free to work on it.
